onto-viewer-web-app/src/main/resources/static/css/main.css

```diff
@@ -142,7 +142,7 @@ h3>b{
   color: #e78747 !important;
 }
 
-.prodDevIndicator {
```

Comment:
I don't see usage of this anywhere in the project. @dasawanaka Can you confirm? If so, we can remove this CSS class.

Reply:
it is used in the development web-app: [link to code](https://github.com/edmcouncil/onto-viewer/blob/961f0c565720e223a4e46480df5833d5ef489212/onto-viewer-web-app/src/main/java/org/edmcouncil/spec/ontoviewer/webapp/util/custom/jsp/tag/TreeRenderTag.java#L26)
It is not used explicitly but class names are created dynamically for the tree

Reply:
Ok, I see. So it should stay like this.